utils/contextutils/context.go

```diff
@@ -23,8 +29,20 @@ const (
 	// TimeReceivedMetadataKey is optional metadata in the gRPC response header that correlates
 	// to the time right after the point cloud was captured.
 	TimeReceivedMetadataKey = "viam-time-received"
+
+	readTimeout            = 5 * time.Second
+	readTimeoutBehindProxy = time.Minute
+	initialReadTimeout     = 1 * time.Second
 )
 
+// ViamDotDir is the directory for Viam's cached files.
+var ViamDotDir string
```

Comment:
The config read timeout vars' presence in this package is acceptable IMO, but I don't _love_ `ViamDotDir` being here. It's only barely related to context creation; do you have any ideas for a better location @cheukt ?

Reply:
utils/env.go maybe? but ya, not an easy var to find a location for

Reply:
Let's do `utils/env.go` if that SGTY @bashar-515 , I like that better. Thanks, @cheukt .